src/panels/lovelace/cards/hui-media-control-card.ts

```diff
@@ -84,106 +105,168 @@ export class HuiMediaControlCard extends LitElement implements LovelaceCard {
 
     const picture = this._image || "../static/images/card_media_player_bg.png";
 
+    const imageStyle = {
+      "background-image": `url(${this.hass.hassUrl(picture)})`,
+      width: `${!this._image ? "50%" : `${this._cardHeight}px`}`,
```

Comment:
I personally find a negative check in a ternary statement confusing. Prefer this:

```suggestion
      width: `${this._image ? `${this._cardHeight}px` : "50%"}`,
```

Reply:
Gotcha. We do Negative checks for Template Results. So I usually follow that mind set. Will update going forward